manifest.appcache

```diff
@@ -1,5 +1,5 @@
-# Random number: 23
-# Random number (or any change) necessary to update client cache.
+# Versions (or any changes) are necessary to update client cache.
+# Version: 1.0
```

Comment:
It doesn't need to be a point version. A single integer that gets incremented by one every time is fine.